index.html

```diff
@@ -46,14 +46,69 @@
             <div class="burger-menu__item"></div>
           </div>
         </div>
-
       </header>
+
+      <nav class="navigation-mobile">
+        <div class="navigation-mobile-header">
+          <a class="logo" href="./index.html">
+            <img src="./images/logo.svg" alt="logo">
+          </a>
+          <div class="burger-menu">
+            <div class="burger-menu__item"></div>
+            <div class="burger-menu__item"></div>
+            <div class="burger-menu__item"></div>
+          </div>
+        </div>
+        <ul class="navigation-mobile__list">
+          <li class="navigation-mobile__item">
+            <a class="navigation-mobile__link" href="#">About the shelter</a>
+          </li>
+          <li class="navigation-mobile__item">
+            <a class="navigation-mobile__link" href="#">Our pets</a>
+          </li>
+          <li class="navigation-mobile__item">
+            <a class="navigation-mobile__link" href="#">Help the shelter</a>
+          </li>
+          <li class="navigation-mobile__item">
+            <a class="navigation-mobile__link" href="#">Contacts</a>
+          </li>
+        </ul>
+      </nav>
+
+      <section class="startscreen">
+        <div class="startscreen-content">
+          <h1 class="startscreen__title">Not only people need a house</h1>
+          <p class="startscreen__text">We offer to give a chance to a little and nice puppy
+            with an extremely wide and open heart. He or she will love
+            you more than anybody else in the world, you will see!
+          </p>
+          <a class="startscreen__link" href="#">
+            Make a friend
+          </a>
+        </div>
+        <div class="startscreen-img">
+          <img class="puppy-img" src="./images/puppy_startscreen_desk.png" alt="puppy">
+        </div>
+      </section>
+
     </div>
 
 
   </div>
+</body>
 
+<script>
+  const burgerMenu = document.querySelectorAll('.burger-menu');
+  const logo = document.querySelector('.logo');
+  const navigationMobile = document.querySelector('.navigation-mobile');
 
-</body>
+  burgerMenu.forEach(function (element) {
+    element.addEventListener("click", function (e) {
+      navigationMobile.classList.toggle('_active');
+      logo.classList.toggle('_active');
```

Comment:
Нам нужно использовать модификаторы.

```suggestion
      navigationMobile.classList.toggle('navigation-mobile--active');
      logo.classList.toggle('logo--active');
```